DungeonWorld/DAO/DataAccesObject.cs

```diff
@@ -1,62 +1,26 @@
 ﻿using DungeonWorld.Model;
 using System;
 using System.Collections.Generic;
-using System.Data.Entity;
 using System.Linq;
-using System.Threading.Tasks;
 
 namespace DungeonWorld.DAO
 {
     public class DataAccesObject
     {
         public static DWModel context = new DWModel();
 
-        public static Race GetRaceById(int id)
-        {
-            
-            return context.Races.Where(r => r.IDRace == id).FirstOrDefault();
-            
+        public static Race GetRaceById(int id) => context.Races.FirstOrDefault(r => r.RaceId == id);
 
-        }
-        public static Race GetRaceIdByName(string name)
-        {
+        public static Race GetRaceIdByName(string name) => context.Races.FirstOrDefault(r => r.RaceName == name);
 
-            return context.Races.Where(r => r.Race_Name == name).FirstOrDefault();
+        public  static Class GetClassById(int id) =>  context.Classes.SingleOrDefault(r => r.IDClass == id);
+        
+        public static Class GetClassIdByName(string name) => context.Classes.FirstOrDefault(r => r.Class_Name == name);
 
+        public static List<string> GetClassNames() => context.Classes.Select(c => c.Class_Name).ToList();
+        
+        public static List<String> GetRaceNames() => context.Races.Select(c => c.RaceName).ToList();
 
-        }
-
-        public  static Class GetClassById(int id)
-        {
-            
-            return  context.Classes
-                               .Where(r => r.IDClass == id)
-                               .SingleOrDefault();
-
-        }
-        public static Class GetClassIdByName(string name)
-        {
-            var query = context.Classes
-                               .Where(r => r.Class_Name == name).FirstOrDefault();
```

Comment:
Słabe łamanie linii. nie stawiaj niepotrzebnych spacji, tabów. Visual/Rider ładnie to formatuje (tak jak był powinno)